cmd/stcli/cmd_gui.go

```diff
@@ -48,7 +48,7 @@ func guiDump(c *cli.Context) {
 	writer := newTableWriter()
 	fmt.Fprintln(writer, "Enabled:\t", cfg.Enabled, "\t(enabled)")
 	fmt.Fprintln(writer, "Use HTTPS:\t", cfg.UseTLS(), "\t(tls)")
-	fmt.Fprintln(writer, "Listen Addresses:\t", cfg.Address(), "\t(address)")
+	fmt.Fprintln(writer, "Listen Addresses:\t", strings.Join(cfg.Addresses(), ","), "\t(address)")
```

Comment:
Should probably call it "addresses" instead of "address" in all key checks.

Reply:
Which key checks are you talking about?

Reply:
The switch statement, `case "address"` should become `case "addresses"` (as well as address -> addresses in all other places)

Reply:
Fine